kazu/steps/ner/hf_token_classification.py

```diff
@@ -67,24 +62,25 @@ def __init__(
         batch_size: int,
         stride: int,
         max_sequence_length: int,
-        labels: list[str],
-        detect_subspans: bool = False,
-        threshold: Optional[float] = None,
+        tokenized_word_processor: TokenizedWordProcessor,
+        keys_to_use: Iterable[str],
         entity_splitter: Optional[NonContiguousEntitySplitter] = None,
-        strip_re: Optional[dict[str, str]] = None,
+        device: Optional[str] = None,
```

Comment:
should the default be `None` or better `cpu`? Or even better it can automatically detect if cuda is available (`"cuda" if torch.cuda.is_available() else "cpu" as a global variable)` )? 
The doc string says defaults to cpu.

Reply:
good point this should be cpu. I don't want to make it automatically detect/use the GPU, as this might have unexpected consequences for the user. For instance, in an environment where the GPU is doing something else, and the user has Kazu running in the background, it might cause all kinds of problems